### ISSUE TITLE
Activity App MVP

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#11545]
# Description

## User Stories

* > As a I want to see all activities so that I'm in full control.

## Value

- Transparency
- Accountability
- Collaboration
- Error resolution

## Acceptance Criteria
- New App `Activities`
- initial: shows all activities of personal (initial default filter)
- Provide filter for Location (Spaces, Shares, Personal)
- Provide filter for time (same as in search)

## Definition of ready
- [ ] Everybody needs to understand the value written in the user story
- [ ] Acceptance criteria have to be defined
- [ ] All dependencies of the user story need to be identified
- [ ] Feature should be seen from an end user perspective
- [ ] Story has to be estimated
- [ ] Story points need to be less than 20

## Definition of done
- Functional requirements
  - [ ] Functionality described in the user story works
  - [ ] Acceptance criteria are fulfilled
- Quality
  - [ ] Code review happened
  - [ ] CI is green (that includes new and existing automated tests)
  - [ ] Critical code received unit tests by the developer
- Non-functional requirements
  - [ ] No sonar cloud issues

## Visual
![image](https://github.com/user-attachments/assets/f48cf83d-aa1e-4c73-bf9d-8eb550564ec6)

**Comment by tbsbdr** [comment]
on hold

**Comment by AlexAndBear** [comment]
WIP is here:
https://github.com/owncloud/web/pull/11564

Not included: 
 - Date Filter (backend implementation needed)
 - Pagination Filter (backend implementation needed)
 - Activity Type (backend implementation needed)

**Comment by dj4oC** [comment]
Before this can be acted on, the following needs to be resolved:

- This issue is explicitly marked "on hold" by a maintainer.
- The linked WIP PR (#11564) excludes several items that need backend implementation (date filter, pagination filter, activity type) or a UX concept (representation of non-file/folder activities like "user added to space"), none of which are confirmed to have progressed since 2024.

Could someone confirm whether this is still on hold, and if not, which of the excluded sub-items (backend-dependent or concept-dependent) have since been unblocked?

🤖 Automated by Claude Code

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
